tests/kernel/usb_drive.pm

```diff
@@ -0,0 +1,58 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: usb_nic
+# Summary: Simple smoke test for testing USB NIC connected to system
```

Comment:
neat picking, the summary should be updated as it tests a drive though

Reply:
Lol nice catch